src/utils/_functions.scss

```diff
@@ -0,0 +1,3 @@
+@function container-size($cell-size, $cell-gap, $container-padding) {
+  @return 7 * $cell-size + 6 * $cell-gap + 2 * $container-padding;
+}
```

Comment:
The best solution for always having 7 days in a week, especially the function taking in multiple parameters,it gives your calendar a great deal of flexibility 🤩